### ISSUE TITLE
AutoApply on customShortcuts

### ISSUE THREAD

**Issue by tbordinat** [kenhyuwa/litepie-datepicker#8]
On mobile, when using customShortcuts, it's impossible to apply changes when autoApply is set to false.

![image](https://user-images.githubusercontent.com/4682556/111613832-3ca78380-87df-11eb-95c0-8f3604d70a71.png)

It could be great to set dates directly and submitting it.

Here is the code of my shortcuts, maybe I missed something :

```js

      return [
        {
          label: '7 derniers jours',
          atClick: () => {
            const date = new Date()
            return [
              new Date(date.setDate(date.getDate() - 7)),
              new Date(),
            ]
          },
        },
// ...
```

**Comment by kenhyuwa** [comment]
I have try, and work with `auto-apply` is `false`

<img width="218" alt="Screen Shot 2021-03-18 at 18 08 13" src="https://user-images.githubusercontent.com/17788063/111617012-29afa600-8815-11eb-9016-f238d4a230b7.png">

this my code
```vue
<litepie-datepicker
  :formatter="formatter"
  :auto-apply="false"
  :shortcuts="shortcuts"
  v-model="dateValue"
>
</litepie-datepicker>
```

**Comment by tbordinat** [comment]
My bad, the problem occurs when it's set to true (default value in my case).

**Comment by kenhyuwa** [comment]
when `auto-apply` is `true`,
if you click shortcut, `v-model` has updated with your selection.
you can changing button `cancel` with new props `footer`, read [here](https://litepie.com/#footer).
Thank you

**Comment by kenhyuwa** [comment]
I'll close this issue. Please open new issues if any related issues.

**Comment by tbordinat** [comment]
With this fix in 1.0.9, it's ok ! 

`Fix bug auto-apply if shortcuts clicked.`
